---
bool succeeded
int8 robot_state
